### PR TITLE
Release v9.0.0

### PR DESCRIPTION
**Pull request by lildude**
Time for a new release, and it's MAJOR release 🎉 

I'm making this a major release as we've refactored `Linguist::Repository` to isolate Rugged usage. Whilst this shouldn't break anything, this is a relatively significant change to the way repos can be read.

### Tasks

- [ ] Gem pushed to RubyGems & GitHub Package Registry
- [ ] Deployed to GitHub.com
- [ ] Grammars deployed to Syntax highlighting service - highlighting won't be 💯 up-to-date until this happens

I'm aiming to make and deploy this release the week of 25 Nov '24.

NOTE: Search results will not reflect changes and new lanuguages until the upstream internal library is updated to pull in the changes from this release and that then deployed to GitHub's Search.

### CODE REVIEW

File: vendor/licenses/git_submodule/vsc-language-1c-bsl.dep.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 name: vsc-language-1c-bsl
-version: 0204d3f7c9d894bf816503252bd3684677d764b3
+version: f6d3a6b7b82de4ea446678cb0a796df2717635db
 type: git_submodule
 homepage: https://github.com/1c-syntax/vsc-language-1c-bsl.git
-license: mit
+license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/vsc-language-1c-bsl.dep.yml:6]:
```suggestion
license: mit
```

File: vendor/licenses/git_submodule/vscode-go.dep.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 name: vscode-go
-version: 725259766181b4be91ed84411d6ac564a5ab2e5a
+version: 3e4432be7a8aa47517fc8d82b268eae3ef391a5b
 type: git_submodule
 homepage: https://github.com/golang/vscode-go
-license: mit
+license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/vscode-go.dep.yml:6]:
```suggestion
license: mit
```

File: vendor/licenses/git_submodule/wgsl-analyzer.dep.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 name: wgsl-analyzer
-version: 8851962fc191aa5ea85a25b0ebd10cb9f70627b3
+version: 5175e187b2d2ca33e8b50ddafb3ba4899473572b
 type: git_submodule
 homepage: https://github.com/wgsl-analyzer/wgsl-analyzer.git
 license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/wgsl-analyzer.dep.yml:6]:
```suggestion
license: mit
```

File: vendor/licenses/git_submodule/vscode-scala-syntax.dep.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 name: vscode-scala-syntax
-version: fb73e8a0bfcd9a3c45f2c1b712e00c35865a9178
+version: 1fa3885da1d7bee4c58feafcb386649d898a43be
 type: git_submodule
 homepage: https://github.com/scala/vscode-scala-syntax
 license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/vscode-scala-syntax.dep.yml:6]:
```suggestion
license: mit
```

**Review by timscharfenort8 — COMMENTED** [review]
first-level heading

**Comment by timscharfenort8** [comment]
```c#

```

**Comment by Scutua** [comment]
Thank you very much

**Review by adnanmohammed573 — COMMENTED** [review]
Addynmj

**Review by adnanmohammed573 — COMMENTED** [review]
Good

**Review by adnanmohammed573 — COMMENTED** [review]
Addd

**Pull request merged**